lib/key/utils.js

```diff
@@ -11,23 +11,9 @@ export function generateSessionKey(algorithm) {
     return openpgp.crypto.generateSessionKey(algorithm);
 }
 
-export function reformatKey(privKey, email = '', passphrase = '') {
-    if (passphrase.length === 0) {
-        return Promise.reject(new Error('Missing private key passcode'));
-    }
```

Comment:
Maybe we want to keep this check? I.e. throw if `!options.passphrase`. The reason being that openpgp.js just returns an unencrypted key in that case, which might lead to a vulnerability. (The same thing goes for `generateKey`, really, but you might want to double check that we always pass a passphrase there before adding the check there.)